src/CMakeLists.txt

```diff
@@ -43,30 +45,30 @@ endif()
 include(shared/sources.cmake)
 include(apps/vpn/cmake/sources.cmake)
 
```

Comment:
This can be called as you like.

Reply:
@bakulf I don't understand what you mean by this comment. Would you mind expanding?